src/Cookie/CookieJar.php

```diff
@@ -256,7 +259,8 @@ private function getCookiePathFromRequest(RequestInterface $request): string
         if ('/' === $uriPath) {
             return '/';
         }
-        if (0 === $lastSlashPos = \strrpos($uriPath, '/')) {
+        $lastSlashPos = \strrpos($uriPath, '/');
+        if (0 === $lastSlashPos || false === $lastSlashPos) {
```

Comment:
but isn't this changing the behavior here?

https://3v4l.org/rZZ4s


related comment: https://github.com/guzzle/guzzle/pull/2677#discussion_r436259529

Reply:
Good one. I've spun up XDebug to verify your assumption. `CookieJarTest::testCookiePathWithEmptySetCookiePath` seems to properly cover this.

Line 253 lets the method return early in case there isn't any `/` in `$uriPath`, so `$lastSlashPos` will never be `false`:

```
if (0 !== \strpos($uriPath, '/')) {
```